src/components/Footer/Footer.js

```diff
@@ -1,5 +1,136 @@
+            <div className="serviceTime">
+              <dl className="timeBox">
+                <dt className="timeTerm">
+                  운영시간 평일 11:00 - 18:00 (토∙일, 공휴일 휴무)
+                  <br />
+                  점심시간 평일 13:00 - 14:00
+                </dt>
+              </dl>
+            </div>
+            <p className="serviceNoti">1:1 문의하기는 앱에서만 가능합니다.</p>
+            <div className="serviceBtnBox">
+              <span className="small">자주 묻는 질문</span>
+            </div>
+          </div>
+          <div className="footerMenu">
+            <div className="menuBox">
+              <div className="menuTitle">
+                이용안내
+                <ul className="menuList">
+                  <li className="memuItem">
+                    <span className="menuLink">검수기준</span>
+                  </li>
+                  <li className="memuItem">
+                    <span className="menuLink">이용정책</span>
+                  </li>
+                  <li className="memuItem">
+                    <span className="menuLink">패널티 정책</span>
+                  </li>
+                  <li className="memuItem">
+                    <span className="menuLink">커뮤니티 가이드라인</span>
+                  </li>
+                </ul>
+              </div>
+            </div>
+            <div className="menuBox">
+              <div className="menuTitle">
+                고객지원
+                <ul className="menuList">
+                  <li className="memuItem">
+                    <span className="menuLink">공지사항</span>
+                  </li>
+                  <li className="memuItem">
+                    <span className="menuLink">서비스 소개</span>
+                  </li>
+                  <li className="memuItem">
+                    <span className="menuLink">스토어 안내</span>
+                  </li>
+                  <li className="memuItem">
+                    <span className="menuLink">판매자 방문접수</span>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="corporationArea">
+          <ul className="termList">
+            <li className="termItem">
+              <span className="termLink">회사소개</span>
+            </li>
+            <li className="termItem">
+              <span className="termLink">인재채용</span>
+            </li>
+            <li className="termItem">
+              <span className="termLink">제휴제안</span>
+            </li>
+            <li className="termItem">
+              <span className="termLink">이용약관</span>
+            </li>
+            <li className="termItem privacy">
+              <span className="termLink bold">개인정보처리방침</span>
+            </li>
+          </ul>
+          <div className="footerSns">
+            <div className="snsBox">
+              <span className="sns" aria-label="인스타그램" />
+              <span className="sns" aria-label="페이스북" />
+              <span className="sns" aria-label="카카오톡" />
```

Comment:
aria-label을 사용하시는 이유가 있을까요?? 
```suggestion
              <span className="sns" > 인스타그램 </span>
```
처럼 해당영역을 임시로 지정해놓을 수 있지 않을까요?